src/main/java/app/task/TaskController.java

```diff
@@ -0,0 +1,72 @@
+package app.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+
+@RestController()
+public class TaskController {
+
+    @Autowired
+    private TaskDao taskDao;
+
+    @InitBinder
+    private void initBinder(WebDataBinder binder) {
+        binder.setValidator(new TaskValidator());
+    }
+
+    @GetMapping("/tasks")
+    public List<Task> getAllTasks() {
+        return taskDao.getAllTasks();
+    }
+
+
+    @GetMapping("/tasks/{taskId}")
+    public Task getTask(
+            @PathVariable int taskId
+    ) {
+        return taskDao.getTask(taskId);
```

Comment:
W metodzie deleteTask sprawdzasz czy istnieje i jezeli nie to zwracasz NOT_FOUND, czemu tutaj tego nie robisz?